woocommerce/woocommerce-gutenberg-products-block
# Add Feedback Prompt in Cart & Checkout blocks sidebar

Fixes #1300.

I couldn't get it to appear after the _Advanced_ section. We could achieve this with some custom CSS using flexbox, but I'm not happy modifying the Gutenberg sidebar styles because we might break unexpected things now or in the future, so I would prefer another solution if anybody knows it or leaving the current implementation if we consider it's good enough.

#### Accessibility


### Screenshots

![imatge](https://user-images.githubusercontent.com/3616980/70520770-59fe9400-1b3e-11ea-8aea-3f0fbff420a1.png)

### How to test the changes in this Pull Request:

1. Create a post with a _Cart_ or _Checkout_ blocks.
2. Verify there is a _Feedback?_ prompt in the sidebar.
